src/TritonRoute/src/TritonRoute.cpp

```diff
@@ -284,7 +288,9 @@ void TritonRoute::readParams(const string& fileName)
           CMAP_FILE = value;
           ++readParamCnt;
         } else if (field == "threads") {
-          MAX_THREADS = atoi(value.c_str());
+          logger_->warn(
+              utl::DRT, 999, "deprecated threads param in params file."
```

Comment:
assign a real number for the message and update messages.txt

Reply:
use etc/FindMessages.tcl > messages.txt to check for ID conflicts